lib/twilio/eventpublisher.ts

```diff
@@ -58,7 +59,7 @@ class EventPublisher extends EventEmitter {
         get() { return isEnabled; },
         set(_isEnabled) { isEnabled = _isEnabled; },
       },
-      _log: { value: options.log },
+      _log: { value: new Log('EventPublisher') },
```

Comment:
Should we still allow an option override here? `options.log || new Log`?

Reply:
We don't need to do that anymore. Actually, we shouldn't do that anymore because we want each class to create their own log instance.